README.md

```diff
@@ -1,5 +1,6 @@
 # Prometheus
 
+[![Gitpod ready-to-code](https://img.shields.io/badge/Gitpod-ready--to--code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/prometheus/prometheus)
```

Comment:
This should go at the end of the list, it's not more important than our CI, downloads etc.

Reply:
Yeah, I was on the fence about that one. You could argue it's more useful for actually clicking on it more often, but yeah, maybe it should be at the end for now.